app/components/Navbar.js

```diff
@@ -11,14 +11,12 @@ export default (props) => {
     <Navbar inverse style={style} className="navbar-fixed-top">
       <Navbar.Header>
         <Navbar.Brand>
-          <a href="#">Gammapy Catalog Browser</a>
+          <a href="#">Gammapy Sky</a>
```

Comment:
How about "Gammapy Web Experiments"?

It's not really a useful webpage at the moment, so we shouldn't call it "Gammapy Sky" in case someone finds it.